framework/fp_vectors/Floats.h

```diff
@@ -71,6 +82,20 @@ typedef SANDSTONE_FP16_TYPE fp16_t;
 #define FLOAT64_MANTISSA_QUIET_NAN_MASK  0x8000000000000u
 #define FLOAT80_MANTISSA_QUIET_NAN_MASK  0x4000000000000000u
 
+#define BFLOAT8_SIGN_BITS        1
+#define BFLOAT8_EXPONENT_BITS    5
+#define BFLOAT8_MANTISSA_BITS    2
+#define BFLOAT8_INF_AT_INPUT_MANTISSA   0b00
+#define BFLOAT8_OVERFLOW_MANTISSA       0b01
```

Comment:
The reference to OCP is removed from Intel's specs.
Overflow in our specs is defined as `00` mantissa (and `01` is for "unspecified" NaN). Should we assume, that this is a mistake in docs (and follow OCP), or should force OVERFLOW==INF, and `01` should be just "impossible" NaN (which is identical with quiet)?